bundles/org.openhab.binding.boschshc/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.classpath
+/.project
```

Comment:
These are excluded at project level and don't need to be excluded in your bundle.

Reply:
👍 Alright, removed the .gitignore file for the bundle